### ISSUE TITLE
version command displays wrong version

### ISSUE THREAD

**Issue by ohkinozomu** [GoogleContainerTools/container-diff#346]
### Information

- container-diff version: 0.16.0(probably)
- Operating system: 
```
$ sw_vers
ProductName:	Mac OS X
ProductVersion:	10.15.7
BuildVersion:	19H2
```

### Steps to reproduce the behavior

```
$ curl -LO https://storage.googleapis.com/container-diff/v0.16.0/container-diff-darwin-amd64 && mv container-diff-darwin-amd64 container-diff && chmod +x container-diff && sudo mv container-diff /usr/local/bin/
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 14.5M  100 14.5M    0     0  17.2M      0 --:--:-- --:--:-- --:--:-- 17.2M
$ container-diff version
v0.15.0 built from git 2db6995
$ container-diff version --short
v0.15.0
```

I think this can be fixed with
https://github.com/GoogleContainerTools/container-diff/blob/v0.16.0/version/version.go#L22 ,
but I'm not releaser, so I shouldn't touch it.

**Comment by ohkinozomu** [comment]
I forgot to mention, I'm grateful and excited about the release of 0.16.0!

**Comment by ohkinozomu** [comment]
I missed the PR https://github.com/GoogleContainerTools/container-diff/pull/345

**Comment by reini-1** [comment]
Any news about that. still wrong version shown.
Thanks in advance.

**Comment by nkubala** [comment]
@ohkinozomu thanks for opening this and @reini-1 thanks for following up. this is fixed now in v0.17.0, so I'm going to close this issue. sorry for the delay!
